scripts/list-languages-in-enterprise.js

```diff
@@ -0,0 +1,100 @@
+const { listAllRepositoriesInOrganization } = require("../util/github");
+
+const getTotalCodeSizeInBytes = (obj) => Object.values(obj).reduce((accumulator, value) => accumulator + value, 0);
+
+module.exports = {
+    description: "list language usage within an enterprise",
+    options: {
+        enterprise: {
+            alias: "e",
+            demandOption: true,
+            describe: "the GitHub enterprise to search for language usage",
+            type: "string",
+        },
+        percent: {
+            alias: "p",
+            default: 25,
+            describe: "what percentage of code is needed to qualify as a \"used\" language",
+            type: "number",
+        },
+    },
+    action: async (octokit, graphql, argv) => {
+        const organdrepo = [];
+        let hasNextPage = true;
+        // workaround to avoid code duplication on next line, allows null initial value which graphql requires
+        // eslint-disable-next-line unicorn/no-null
+        let endCursor = null;
+        const languagesInEnterprise = {};
+
+        while (hasNextPage) {
+            const enterpriseInfo = await graphql(`
+                query ListOrgs {
+                  viewer {
+                    login
+                  }
+                  enterprise(slug: "${argv.enterprise}") {
+                    id
+                    organizations(
+                        first: 100
+                        after: ${endCursor}
+                      ) {
+                      nodes {
+                        url
+                      }
+                      pageInfo {
+                        startCursor
+                        hasPreviousPage
+                        hasNextPage
+                        endCursor
+                      }
+                    }
+                    slug
+                    name
+                  }
+                }
+            `);
+
+            hasNextPage = enterpriseInfo.enterprise.organizations.pageInfo.hasNextPage;
+            // workaround to avoid code duplication on next line, allows null initial value which graphql requires
+            // eslint-disable-next-line prefer-template, quotes
+            endCursor = '"' + enterpriseInfo.enterprise.organizations.pageInfo.endCursor + '"';
```

Comment:
I don't understand the eslint comment, isn't this templated version equivalent? 

```suggestion
            endCursor = `"${enterpriseInfo.enterprise.organizations.pageInfo.endCursor}"`;
```

```
> const a = null;
> '"' + a + '"' === `"${a}"`
true
```